db/migrate/20160921160155_add_origin_to_fact_values.rb

```diff
@@ -0,0 +1,8 @@
+class AddOriginToFactValues < ActiveRecord::Migration
+  def change
+    add_column :fact_values, :origin, :string, :limit => 255, :default => 'N/A'
```

Comment:
Good call, I created this column because I assumed we'd have problems distinguishing fact names with the same name for distinct config management systems. I'll make `FactName::Puppet` in this PR too and remove the migrations